scripts/build-env.js

```diff
@@ -6,6 +6,7 @@ switch (process.env.BUILD_MODE) {
   case 'release':
     sdkVersion = lernaJson.version
     break
+  case 'canary':
```

Comment:
what about reflecting it in [BuildMode](https://github.com/DataDog/browser-sdk/blob/main/packages/core/src/boot/init.ts#L40) too?